pyartcd/pyartcd/pipelines/prepare_release_konflux.py

```diff
@@ -958,19 +1092,11 @@ async def prepare_release(
     runtime: Runtime,
     group: str,
     assembly: str,
+    date: str,
     build_repo_url: Optional[str],
     shipment_repo_url: Optional[str],
 ):
-    job_url = os.getenv('BUILD_URL')
```

Comment:
i'm not sure about this.. 
I find it better to get all env vars in the click command and then pass it to pipeline. It makes mocking and testing easier, the external inputs are gathered together and passed ..

Reply:
I didn't have to change tests, looks like no overhead? OTOH this makes code cleaner imo but it's easy to revert if you have strong opinions